feedback.md

```diff
@@ -0,0 +1,19 @@
+# Startrly
+## What We're Looking For
+
+Feature | Feedback
+--- | ---
+**General** |
+Small commits with meaningful commit messages |
+Answered comprehension questions |
+**Code Requirements** |
+Only limited changes to the HTML |
+Page normalized |
```

Comment:
Changes should be applied to CSS, not HTML.

Reply:
They can add classes to the HTML